src/ontology/mondo-ingest.Makefile

```diff
@@ -413,18 +413,17 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl --add-prefixes config/context.json \
 			 remove --term "http://purl.obolibrary.org/obo/mondo#ABBREVIATION" --preserve-structure false -o $@
 
-tmp/merged.db: tmp/merged.owl
+tmp/%.db: tmp/%.owl
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
-	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	RUST_BACKTRACE=full semsql make tmp/$*.db -P config/prefixes.csv
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
 
-mondo-ingest.db: tmp/merged.db
+merged.db: tmp/merged.db
```

Comment:
## Rename: `mondo-ingest.db` --> `merged.db`
Justification: (i) `merged.db` is a literal copy of `tmp/merged.db`, which comes from `merged.owl`, (ii) Synchronization pipeline has need of `mondo-ingest.db`, which comes from `mondo-ingest.owl`.

Personally I don't even thing we need `merged.db`. We have goals which depend on `tmp/merged.db` (which is exactly the same) but no goals that depend on `merged.db`.

Reply:
If merged.db is not needed anywhere can we renamed it mondo-ingest-merged.db, because I think this file is a file that Chris was asking for in the past (mondo+all ingests) merged together.